internal/operators/cnv/cnv_operator.go

```diff
@@ -37,8 +40,14 @@ var Operator = models.MonitoredOperator{
 
 // NewCNVOperator creates new instance of a Container Native Virtualization installation plugin
 func NewCNVOperator(log logrus.FieldLogger) *operator {
+	var cfg Config
```

Comment:
Usually, we transfer the components configurations from `main.go` , so there won't be any other processes just for that

Reply:
That was deliberate design decision to decouple specific OLM plugins (and their configuration) from the core code; this way there is less changes required in the core while adding new plugins.

Reply:
@filanov WDYT?

Reply:
I am leveraging this to customize CNV [subscription](https://github.com/openshift/assisted-service/pull/1421) and seems to be consistent with what OCS is [doing](https://github.com/openshift/assisted-service/blob/master/internal/operators/ocs/ocs_operator.go#L30).

Reply:
i'm a bit out of context but usually if we a configuration form env vars we are doing it at main.
in the main it will be clearer what configurations are supported.

Reply:
I wanted to make it operator plugin's responsibility to configure oneself using whatever method it wants but
if you think it is a better approach to pass the configuration from main, I can use existing `operators.Options` (https://github.com/openshift/assisted-service/blob/2f36f8a677d2b8d28d4ff1318de9513b28c822dd/internal/operators/builder.go#L25).

Reply:
if it was a status configuration that we don't plan to override then you could create a function like `NewCNVOperatorWithDefailtConfig` but if we want to give the operation to override and we get it from env vars it clearer to see where all the configuration is located.

Reply:
It's rather the latter case. I've updated the code to use config loaded at the top level.